EIPS/eip-4788.md

```diff
@@ -75,6 +75,8 @@ Validity of the parent beacon block root is guaranteed from the consensus layer,
 
 When verifying a block, execution clients **MUST** ensure the root value in the block header matches the one provided by the consensus client.
 
+For a genesis block with no existing parent beacon block root the 32 zero bytes are used as a root placeholder.
```

Comment:
```suggestion
For a genesis block with no existing parent beacon block root, the root consisting of 32 zero bytes should be used in the block header.
```

Reply:
Ah, now the old version still made it into the merged spec, was this intended? 🤔